flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetRecordFormat.java

```diff
@@ -42,7 +43,17 @@
 
 import static org.apache.flink.util.Preconditions.checkArgument;
 
-/** A reader format that reads individual Avro records from a Parquet stream. */
+/**
+ * A reader format that reads individual Avro records from a Parquet stream. This class leverages
+ * {@link ParquetReader} underneath. Developer should make sure the parquet files can be worked with
+ * provided avro schema and take care of any further compatibility issue.
+ *
+ * <p>It is recommended to use the factory class {@link AvroParquetReaders} which is capable to
```

Comment:
Suggestion:

```
For instantiation, it is recommended to use the factory class  {@link AvroParquetReaders}. It is capable of creating 
versions of {@link AvroParquetRecordFormat} that can work with {@link GenericRecord GenericRecords}, {@link 
org.apache.avro.specific.SpecificRecord SpecificRecords}, or {@link org.apache.avro.reflect.ReflectData reflect 
records}.
```
But honestly, I would remove the second sentence, because this is that the user will see in the API of the AvroParquetReaders class.